whylog_vim/gui/files_manager.py

```diff
@@ -0,0 +1,28 @@
+import vim
+from whylog_vim.gui.vim_commander import VimCommander
+
+
+class FilesManager(object):
+
+    @staticmethod
+    def get_files_window_id(file_name):
+        for window_id, window in enumerate(vim.windows):
+            if window.buffer.name.endswith(file_name):
+                return window_id + 1
+        return None
+
+    @classmethod
+    def is_file_open(cls, file_name):
+        if FilesManager.get_files_window_id(file_name) is not None:
```

Comment:
Maybe write a boolean function that checks if file is open? window_id is not needed here.